# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: b668d0c685e8
# Full Hash: b668d0c685e8d0bbcd3bf5b1856389ca41be31f3
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-09 04:16:57
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 5: Exempt third-party cookie blocking if the channel is on the exception list. r=cookie-reviewers,bvandersloot,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223384
# ==============================================================================

diff -r 08acd8256071 -r b668d0c685e8 dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Wed Jan 08 19:25:59 2025 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Wed Jan 08 19:25:59 2025 +0000
@@ -36,6 +36,7 @@
                    bool thirdPartyContext,
                    bool partitionForeign,
                    bool usingStorageAccess,
+                   bool isOn3PCBExceptionList,
                    bool matchName,
                    nsString name,
                    nsCString path,
@@ -46,6 +47,7 @@
                    bool thirdPartyContext,
                    bool partitionForeign,
                    bool usingStorageAccess,
+                   bool isOn3PCBExceptionList,
                    nsString name,
                    nsString value,
                    bool session,
@@ -60,6 +62,7 @@
                       bool thirdPartyContext,
                       bool partitionForeign,
                       bool usingStorageAccess,
+                      bool isOn3PCBExceptionList,
                       nsString name,
                       nsString path,
                       bool partitioned,